tools/ci_build/github/linux/docker/Dockerfile.manylinux2014_rocm

```diff
@@ -0,0 +1,43 @@
+# TODO unify this with Dockerfile.manylinux2014_cuda10_2
+
+FROM rocm/pytorch:rocm4.1.1_centos7_py3.6_pytorch
```

Comment:
Which cuda version does it use?

Reply:
It's not CUDA actually. It's ROCm 4.1 - the equivalent software stack targeting AMD GPUs. The stack is open-source (see [here](https://rocmdocs.amd.com/en/latest/)). The wheel links to the ROCm libraries but does not redistribute them.

Reply:
So would you please update the comments accordingly?